datahub-web-react/src/app/entity/glossaryTerm/GlossaryTermEntity.tsx

```diff
@@ -45,7 +53,51 @@ export class GlossaryTermEntity implements Entity<GlossaryTerm> {
 
     getEntityName = () => 'Glossary Term';
 
-    renderProfile: (urn: string) => JSX.Element = (_) => <GlossaryTermProfile />;
+    renderProfile = (urn) => {
+        return (
+            <EntityProfile
+                urn={urn}
+                entityType={EntityType.GlossaryTerm}
+                useEntityQuery={useGetGlossaryTermQuery as any}
+                useUpdateQuery={useUpdateGlossaryTermMutation as any}
+                tabs={[
+                    {
+                        name: 'Schema',
+                        component: SchemaTab,
+                    },
+                    {
+                        name: 'Related Entities',
+                        component: GlossaryRelatedEntity,
+                    },
+                    {
+                        name: 'Related Terms',
+                        component: GlossayRelatedTerms,
+                    },
+                    {
+                        name: 'Properties',
+                        component: GlossaryTermPropertyTab,
```

Comment:
does this need to be GlossaryTerm specific? I think we can use the general PropertyTab right?

Reply:
It differs in https://github.com/linkedin/datahub/blob/34c27f076bb7c4c16a295292cee17a7d07fe6165/datahub-web-react/src/app/entity/shared/tabs/Properties/PropertiesTab.tsx#L49 and https://github.com/saxo-datagov/datahub/blob/62d3b692df21b3a748c0f1012bf7ea6bb65bf093/datahub-web-react/src/app/entity/glossaryTerm/profile/GlossaryTermPropertyTab.tsx#L48

Reply:
Got it- we should use `getOverridePropertiesFromEntity` to patch this up then. See how we do this for externalUrl for datasets:

https://github.com/linkedin/datahub/blob/master/datahub-web-react/src/app/entity/dataset/DatasetEntity.tsx#L177

Reply:
yes tanks for ref, done